### ISSUE TITLE
Graph.fetch breaks graph

### ISSUE THREAD

**Issue by kokorin** [clientIO/joint#113]
joint.dia.Graph does'n redefine *parse* method, so if you want to fetch joint.dia.Graph from server you can't just call *fetch* method.

When fetch is done graph.get("cells") is an Array, not joint.dia.GraphCells

**Comment by kokorin** [comment]
Code below makes *fetch* method work
```
            joint.dia.Graph.prototype.parse = function (json, opt) {
                this.fromJSON(json, opt);
                return this;
            };

            joint.dia.Graph.prototype.set = function (key, val, options) {
                if (this == key) return this;
                Backbone.Model.prototype.set.apply(this, arguments);
            }
```

**Comment by kumilingus** [comment]
Would this change only work for you?

     joint.dia.Graph.prototype.parse = function (json, opt) {
           this.fromJSON(json, opt);
           return null;
     };

**Comment by kumilingus** [comment]
Fixed in c55fc4099585b2afe93bd9d6c8fe96f6c9cdafd7.
